code/src/index.css

```diff
@@ -1,13 +1,133 @@
+* {
+  margin: 0; 
+  padding: 0;  
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: content-box;
+  -webkit-font-smoothing: antialiased; 
+  text-rendering: optimizeLegibility;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
+  font-family: -apple-system, BlinkMacSystemFont, Helvetica;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #181818;
+  color: #fff;
+}
+
+.container {
+  margin: 2rem;
+}
+
+a {
+  text-decoration: none;
+  font-size: 14px;
+  line-height: 14px;
+}
+
+.artist-name {
+  color: #a0a0a0;
+  font-size: 14px;
+}
+
+.album-name a {
+  font-weight: 700;
+  color: #ffffff;
+}
+
+.album-name {
+  margin: 10px 0;
+}
+
+.album {
+  display: flex;
+  flex-direction: column;
+  max-width: 70vw;
+  padding: 15px;
+}
+
+.flex-container {
+  margin: 2rem auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.artist-container {
+  display: flex;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.image-container {
+  width: 100%;
+  position: relative;  
+}
+
+.icons-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  align-items: center;
+  justify-items: center;
+  justify-content: center;
+  width: 100%
+}
```

Comment:
was missing some transition/animation for the hover effect on these 😄 , so they don't just hop in and out of existence - a little abrupt in my opinion 😛